staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
+  - single-numa-node
+  - isolate-thread-policy
+zones:
+  node-0:
+    type: Node
```

Comment:
I'm assuming "Node" here means NUMA node not k8s Node?  I think we need to pick a better word

Reply:
yes, it's not a worker node, it's NUMA node. I agree naming could be better.

Reply:
Mm, "node" is quite overloaded term here. Don't know what would be better, NumaNode, MemoryNode, MemoryZone... 🤔

Reply:
NUMANode is at least unambiguous - is it too precise?

Reply:
What about just "NUMA"?

Reply:
Updated this to NUMA in the example but it is important to note here that the this type is populated by an exporter component such as Node Feature Discovery that creates CRs corresponding to nodes in the cluster.